apps/electron/package.json

```diff
@@ -14,13 +14,14 @@
     "make-macos-arm64": "electron-forge make --platform=darwin --arch=arm64",
     "make-macos-x64": "electron-forge make --platform=darwin --arch=x64",
     "make-windows-x64": "electron-forge make --platform=win32 --arch=x64",
-    "postinstall": "ELECTRON_RUN_AS_NODE=1 electron scripts/update-electron-vendors.mjs"
+    "postinstall": "ELECTRON_RUN_AS_NODE=1 electron scripts/update-electron-vendors.mjs && yarn workspace @affine/octobase-node build"
```

Comment:
here we run `cargo build` but all of CI doesn't have such that command

Reply:
I will remove this and let it only run in CI instead